docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,83 @@
+# Bot comparison
+
+This page explains the key differences between Renovate and Dependabot.
+
+This is not meant to be a “versus” or anti-Dependabot page, and it's not marketing.
+It should be as objective as possible, to help users decide which tool is best for them.
+
+If you see anything wrong on this page, please let us know by creating a [Discussion](https://github.com/renovatebot/renovate/discussions), or edit this page with a PR.
+
+## Table
+
+| Name of feature            | Dependabot                                                                                                                                                                   | Renovate                                                                                                 |
+| -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Dependency Dashboard       | No                                                                                                                                                                           | Yes                                                                                                      |
+| Grouped updates            | Yes, create [`groups`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) manually     | Yes, use community-provided groups, or create your own                                                   |
+| Monorepo support           | Unknown                                                                                                                                                                      | Yes                                                                                                      |
```

Comment:
What does `Monorepo support` entail? Can this be elaborated in terms of what renovate supports?

Reply:
Ah, reading further down, monorepo refers here to out of the box upgrade groups

Reply:
Thank you for the explanation, I reworded the table and the sentence about monorepos.

Reply:
I think that the explanation is a bit misleading still. 

Rather than `Upgrades common monorepo packages at once` 

I'd reword it as "Out of the box configuration for common monorepo packages" 

Dependabot can support this using the groups configuration so stating that it is not supported is misleading